### ISSUE TITLE
allow shortcut keys in stand-alone message window

### ISSUE THREAD

**Issue by t4k** [wshanks/tbkeys#77]
over [in the tb forums](http://forums.mozillazine.org/viewtopic.php?f=39&t=2866097&start=15) some folks were working on a shortcut to toggle the show/hide remote content option on messages when the default is set to hide remote content

contributor **morat** has figured out how to add the ability for a message window to respond to shortcuts; I'm wondering if that can be integrated into this extension for everyone; I'd really like this same functionality for myself without resorting to one-off modification

**Comment by wshanks** [comment]
Yes, adding keys to the message window would be good. Do you think it should have its own set of bindings like the compose window (like in morat's suggested changes)? Or should it share bindings with the main window?

**Comment by t4k** [comment]
I would say that it should behave the same as shortcuts in the Message Pane view, as a message window is pretty much the same thing (minus the message list), right? I don't have any technical understanding of this currently.

**Comment by morat523035** [comment]
The message window should have its own set of bindings.

All the tab and calendar functions ( e.g. CloseTabOrWindow ) are not defined in the message window. Some commands do nothing ( e.g. cmd_toggleMessagePane ) in the message window. Many addons are only supported in the main window.
